bin/tractor_load.py

```diff
@@ -38,21 +38,21 @@ def write_index_cluster_files(schema,table=None):
     outname= '%s_index' % table
     rem_if_exists(outname)
     fin=open(outname,'w')
-    for key in list(indexes.keys()): 
+    for key in indexes: 
         for coln in indexes[key]: 
             if coln == 'radec': fin.write('CREATE INDEX q3c_%s_idx ON %s (q3c_ang2ipix(ra, dec));\n' % (key,key))
             else: fin.write('CREATE INDEX %s_%s_idx ON %s (%s);\n' % (key,coln,key,coln)) 
     fin.close()   
-    print(("wrote index file: %s " % outname))
+    print("wrote index file: %s " % outname)
```

Comment:
I know this is much more work, and probably a separate PR, but you should really be using `format()`.  In this case `"wrote index file: {0}".format(outname)`